diesel/src/connection/instrumentation.rs

```diff
@@ -242,10 +242,11 @@ impl<'a> InstrumentationEvent<'a> {
 /// More complex usages and integrations with frameworks like
 /// `tracing` and `log` are supposed to be part of their own
 /// crates.
-pub trait Instrumentation: Send + 'static {
+pub trait Instrumentation: Downcast + Send + 'static {
```

Comment:
I'm not sure if that's a breaking change or not. 
On the one hand it adds a new super trait without a wild card impl for all types on the other hand `Downcast` is implemented for all `T: Any` and any is implemented by the compiler for all types that are `'static` (we already have that bound) so it's likely not breaking?

Reply:
> on the other hand Downcast is implemented for all T: Any and any is implemented by the compiler for all types that are 'static (we already have that bound)

Yes, that's why I thought that it was very unlikely to break anyone 😀